src/Sentry.Unity/ScriptableSentryUnityOptions.cs

```diff
@@ -228,10 +231,11 @@ private void HandlePlatformRestrictions(SentryUnityOptions options, IApplication
                     options.BackgroundWorker = new WebBackgroundWorker(options, SentryMonoBehaviour.Instance);
                 }
 
+                // Disable offline caching regardless whether is was enabled or not.
+                options.CacheDirectoryPath = null;
```

Comment:
I don't get why we should do this now.

Reply:
On "unknown" platforms like Nintendo Switch, we need to turn off any and all file IO as to not crash.